src/Raven.Client/Json/Converters/LazyStringValueJsonConverter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Sparrow.Json;
+
+namespace Raven.Client.Json.Converters
+{
+    internal sealed class LazyStringValueJsonConverter : RavenJsonConverter
+    {
+        public static readonly LazyStringValueJsonConverter Instance = new LazyStringValueJsonConverter();
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            writer.WriteValue(value);
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            if (!(reader is BlittableJsonReader blittableReader))
+            {
+                throw new SerializationException($"{nameof(BlittableJsonReader)} must to be used for convert to {nameof(BlittableJsonReaderObject)}");
+            }
+
+            if (blittableReader.Value == null)
+            {
+                return null;
+            }
+
+            //Todo It will be better to change the reader to set the value as LazyStringValue 
```

Comment:
What if the value is _already_ `LazyBlittableString` ?

Reply:
https://github.com/haludi/ravendb/blob/f13bbd5c65c488aa7c227f6de329e735570a3d44/src/Raven.Client/Json/BlittableJsonReader.cs#L205

The reader convert all the serialized string types to string before it set the value